src/fides/api/ops/service/messaging/message_dispatch_service.py

```diff
@@ -112,7 +113,7 @@ def dispatch_message_task(
 def dispatch_message(
     db: Session,
     action_type: MessagingActionType,
-    to_identity: Optional[Identity],
+    to_identity: Optional[Union[Identity, TestMessage]],
```

Comment:
we can just re-use the `Identity` class for this, since in our `post(MESSAGING_TEST` endpoint, we have the message service type set to whatever has been configured in `CONFIG.notifications.notification_service_type`. So it's OK if both identity types are provided in the body of the test endpoint. The messaging service will handle errors if the appropriate identity has not been provided.

In addition, the `Identity` class already take care of validating format of phone number.

Reply:
Ah, when I initially looked at `Identity` I saw it was inheriting from `BaseSchema` which I was thinking was for a sqlalchemy modes, but not I see it isn't. I'll change it to use this instead.